src/main.cpp

```diff
@@ -1387,17 +1387,20 @@ unsigned int static GetNextWorkRequired_DIGI(const CBlockIndex* pindexLast, cons
     return bnNew.GetCompact();
 }
 
-
+#include "diff_delta.h"
 unsigned int static GetNextWorkRequired(const CBlockIndex* pindexLast, const CBlockHeader *pblock)
 {
+	static int nDeltaSwitchover = fTestNet ? 100 : 213500;
```

Comment:
Yeah we could do it for new codebase, probably no point in doing it for
old codebase at this point I guess.

> In src/main.cpp:
> 
> >  unsigned int static GetNextWorkRequired(const CBlockIndex* pindexLast, const CBlockHeader *pblock)
> >  {
> > +	static int nDeltaSwitchover = fTestNet ? 100 : 213500;
> 
> I like this, can we expand this to all diff algo's? nDGW3Switchover,
> nDigiSwitchover, nKGWSwitchover...
> 
> —
> Reply to this email directly or view it on GitHub.
> 
>